benchmarks/benchmark1.ipynb.raw.html

```diff
@@ -4045,60 +4065,15 @@ <h2 id="Results">Results<a class="anchor-link" href="#Results">&#182;</a></h2>
-<div class="prompt input_prompt">In&nbsp;[6]:</div>
-<div class="inner_cell">
-    <div class="input_area">
-<div class=" highlight hl-ipython3"><pre><span></span><span class="o">%%</span><span class="k">HTML</span>
-&lt;div id=&quot;disqus_thread&quot;&gt;&lt;/div&gt;
-&lt;script type=&quot;text/javascript&quot;&gt;
-    /* * * CONFIGURATION VARIABLES * * */
-    // Required: on line below, replace text in quotes with your forum shortname
-    var disqus_shortname = &#39;chimad-phase-field&#39;;
-    
-    /* * * DON&#39;T EDIT BELOW THIS LINE * * */
-    (function() {
-        var dsq = document.createElement(&#39;script&#39;); dsq.type = &#39;text/javascript&#39;; dsq.async = true;
-        dsq.src = &#39;//&#39; + disqus_shortname + &#39;.disqus.com/embed.js&#39;;
-        (document.getElementsByTagName(&#39;head&#39;)[0] || document.getElementsByTagName(&#39;body&#39;)[0]).appendChild(dsq);
-    })();
-&lt;/script&gt;
-&lt;noscript&gt;Please enable JavaScript to view the &lt;a href=&quot;https://disqus.com/?ref_noscript&quot; rel=&quot;nofollow&quot;&gt;comments powered by Disqus.&lt;/a&gt;&lt;/noscript&gt;
-</pre></div>
-
-</div>
-</div>
-</div>
-
-<div class="output_wrapper">
-<div class="output">
-
-
-<div class="output_area"><div class="prompt"></div>
-
-<div class="output_html rendered_html output_subarea ">
-<div id="disqus_thread"></div>
-<script type="text/javascript">
-    /* * * CONFIGURATION VARIABLES * * */
-    // Required: on line below, replace text in quotes with your forum shortname
-    var disqus_shortname = 'chimad-phase-field';
-    
-    /* * * DON'T EDIT BELOW THIS LINE * * */
-    (function() {
-        var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
-        dsq.src = '//' + disqus_shortname + '.disqus.com/embed.js';
-        (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
-    })();
-</script>
-<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript" rel="nofollow">comments powered by Disqus.</a></noscript>
-</div>
-
+<div class="cell border-box-sizing text_cell rendered">
+<div class="prompt input_prompt">
 </div>
+<div class="inner_cell">
+<div class="text_cell_render border-box-sizing rendered_html">
+<h2 id="Feedback">Feedback<a class="anchor-link" href="#Feedback">&#182;</a></h2><p>Feedback on this benchmark problem is always helpful and actively encouraged. If you have ideas about how the benchmark might be improved or other issues, please feel free to chat with the <a href="https://pages.nist.gov/chimad-phase-field/community/">CHiMaD phase field communtity</a>, via the <a href="https://gitter.im/usnistgov/chimad-phase-field">Gitter chat channel</a>. Alternatively, contact us with an <a href="https://pages.nist.gov/chimad-phase-field/mailing_list/">email</a>. Also, feel free to file an <a href="https://github.com/usnistgov/chimad-phase-field/issues/new">issue on GitHub</a>.</p>
```

Comment:
- "CHiMaD phase field community," not "CHiMaD phase field communtity."
- This text is phrased with the expectation that something is *wrong* with the definition, and solicits corrections. It is preferable to encourage users who may have difficulty to ask us for help. If a user is confident they've found a problem, they will let us know. Otherwise, let's posture ourselves as the experts. Suggested text follows.

> Feedback on this benchmark problem is appreciated. If you have questions, comments, or seek clarification, please contact the [CHiMaD phase field community](https://pages.nist.gov/chimad-phase-field/community/) through the  [Gitter chat channel](https://gitter.im/usnistgov/chimad-phase-field) or by [email](https://pages.nist.gov/chimad-phase-field/mailing_list/). If you found an error, please file an [issue on GitHub](https://github.com/usnistgov/chimad-phase-field/issues/new).